runner/entrypoint.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+if [ -z "${GITHUB_URL}" ]; then
+  echo "Working with public GitHub" 1>&2
+  GITHUB_URL="https://github.com"
```

Comment:
I think we need a trailing `/` here

Reply:
True, to make runner work with older manager